cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,71 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+const faker = require("faker");
 
-describe('Settings page', () => {
-  before(() => {
 
-  });
+import HomePageObject from "../support/pages/home.pageObject";
+import SettingsUserAccount from "../support/pages/settings.pageObject";
+import SignInPageObject from "../support/pages/signIn.pageObject";
 
-  beforeEach(() => {
+const homePage = new HomePageObject();
+const settings = new SettingsUserAccount();
+const signInPage = new SignInPageObject();
+let user;
+
+const testData = {
+  newUsername: faker.name.firstName(),
+  newEmail: faker.internet.email(),
+  newBio: faker.lorem.paragraph(),
+  newPassword: 'goodjob2023!',
+};
+
+
+describe('Settings page', () => {
+  // before(() => {})
 
+  beforeEach(() => {
+    cy.task('db:clear');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    cy.login(user.email, user.username, user.password);
+    cy.visit('/settings');
+    });
   });
 
   it('should provide an ability to update username', () => {
-
+    settings.fillUsernameField(testData.newUsername);
+    settings.clickOnUpdateSettingsBtn();
+    settings.assertNewUsername(testData.newUsername.toLowerCase());
+    
   });
 
   it('should provide an ability to update bio', () => {
-
+    settings.fillUserBioField(testData.newBio);
+    settings.clickOnUpdateSettingsBtn();
+    settings.assertNewBio(testData.newBio);
   });
 
   it('should provide an ability to update an email', () => {
```

Comment:
you do not test here, that you email address is changed

Reply:
fixed